content/about/sidebars.ts

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  about: [
+    {
+      type: 'category',
+      label: 'How It Works',
+      collapsed: false,
+      items: [
+        'about-iota/an-introduction-to-iota',
+        'about-iota/tangle',
+        'about-iota/data-transfer',
+        'about-iota/value-transfer',
+        'about-iota/why-is-iota-feeless',
+        'about-iota/coordinator',
+        'about-iota/roadmap-to-decentralization',
+        'about-iota/messages',
+        'about-iota/energy-efficiency',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'IOTA Token',
+      collapsed: true,
+      items: ['iota-token/buying-iota', 'iota-token/staking-iota'],
+    },
+    {
+      type: 'category',
+      label: 'Media Library',
+      collapsed: true,
+      items: [
+        'media-library/infographics',
+        'media-library/project-presentations',
+        'media-library/videos',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Future of IOTA',
+      collapsed: true,
+      items: [
+        'future/future-of-iota',
+        {
+          type: 'category',
+          label: 'Smart Contracts',
+          collapsed: true,
+          items: [
+            'future/smart-contracts/smart-contracts-introduction',
+            'future/smart-contracts/smart-contracts-VM-and-languages',
+            'future/smart-contracts/smart-contracts-validators',
+            'future/smart-contracts/smart-contracts-chains',
+            'future/smart-contracts/smart-contracts-dapps',
+            'future/smart-contracts/smart-contracts-consensus',
+            'future/smart-contracts/smart-contracts-tangle',
+            'future/smart-contracts/smart-contracts-community-tutorials',
+          ],
+        },
+        'future/shimmer',
+        'future/assembly',
+        'future/role-of-iota',
+        'future/dust-protection',
+        'future/nfts',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Use Cases',
```

Comment:
This may work for the `Use IOTA` section

Reply:
Ok. Let's move it then. We may need to check all this docs again tho. One day we need to have a call where we go over the whole nativ wiki content. 😅